building-server-processdb.py

```diff
@@ -98,7 +100,13 @@ def initDB(city, conf, scoref):
             if len(geoms) == 0:
                 continue
 
-            tileId = nextId()
+            # if there is too much features in the current tile, it is subdivided
+            # in subtiles. The tiles with the highest scores go into the parent tile
+            # and the ones with the lowest scores go into the children tiles.
+            # There is no mention of score into the subdivision process because
+            # we use the fact that they are already sorted in descending order
+            # (done in score_for_polygon function of database.py)
+            tileId = nextId() # id of the next tile (to be genereted to have the right number of feature per tile)
```

Comment:
> to be genereted to have the right number of feature per tile

This is incorrect. The nextId() function generates a unique id for our tiles, it has nothing to do with the number of features per tiles.